data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/resistance/resistance_active_on_defense.cfg

```diff
@@ -0,0 +1,33 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [resistance]active_on=
+##
+# Actions:
+# Give all units resistance to all damage types with a value of 50
+# Attack each other
+##
+# Expected end state:
+# The damage from the attack is reduced by 50% for the attacker
+# The damage from the attack is not for the defenser
```

Comment:
Same typo. However, in this test this line matches the test, and line 7 incorrectly documents the ability.

Reply:
Updated.